app/javascript/components/molecules/Loading.js

```diff
@@ -1,11 +1,22 @@
 import React from 'react'
 import Spinner from '../atoms/Spinner'
 
-const Loading = ({ children, isLoading }) => {
+const Loading = ({
+  children,
+  isLoading = false,
+  renderChildrenWhileLoading = true,
+  loaderViewHeight = null
+}) => {
+  const loaderHeightWrapperStyle = { ...(loaderViewHeight && { height: loaderViewHeight }) }
+  const renderChildren = !isLoading || renderChildrenWhileLoading
   return (
     <div className={'loading-panel' + (isLoading ? ' loading' : '')}>
-      {isLoading ? <Spinner /> : null}
-      {children}
+      {isLoading && (
+        <div style={loaderHeightWrapperStyle}>
+          <Spinner />
```

Comment:
could we use the <Loading/> pattern here too? might have same issue with height but would match what we do elsewhere

Reply:
@akegan haha so look at the file name for this...

Turns out I was wrong about <Loading />, it still renders the child view even if loading is true, so I modified this file to accept a prop to change that behavior.

Reply:
😂 🤦‍♀️